jose/Jwe.ml

```diff
@@ -0,0 +1,37 @@
+(** {{: https://tools.ietf.org/html/rfc7516 } Link to RFC } *)
+
+type aad
+(** Additional Authentication Data *)
+
+type protected
```

Comment:
what are these 2 types doing here? how are they being used?

Reply:
As we said on Discord, this implementation was not supposed to be added here but I failed with git so this is just a stub. Resolving this conversation.